settings.gradle.kts

```diff
@@ -204,9 +204,11 @@ include(":instrumentation:dropwizard:dropwizard-metrics-4.0:javaagent")
 include(":instrumentation:dropwizard:dropwizard-views-0.7:javaagent")
 include(":instrumentation:dropwizard:dropwizard-testing")
 include(":instrumentation:elasticsearch:elasticsearch-rest-common:javaagent")
+include(":instrumentation:opensearch:opensearch-rest-common:javaagent")
```

Comment:
it looks like we may not need a separate `common` artifact, generally we extract those out when there are multiple versions which can share some common code

Reply:
I expect OpenSearch 3.0 to have breaking changes, so we will need multiple versions. Would you prefer to introduce that later?

Reply:
makes sense, it's ok to keep the `common` artifact